src/resources/views/inc/sidebar.blade.php

```diff
@@ -6,11 +6,10 @@
         <!-- Sidebar user panel -->
         <div class="user-panel">
           <div class="pull-left image">
-            <img src="https://placehold.it/160x160/00a65a/ffffff/&text={{ mb_substr(Auth::user()->name, 0, 1) }}" class="img-circle" alt="User Image">
+            <br>
```

Comment:
I agree, in principle, with removing the call to placehold.it but I'm not sure replacing it with a break is the best solution.